### PR TITLE
Ks sk919 translations changing php pversion

### PR DESCRIPTION
**Pull request by Kazarus4**
CCC quality check done
Links checked with gitpod
all translations done

Need proofreading in EN and DE version before PR.

### CODE REVIEW

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/home.it-it.md
  | SyntaxError: invalid syntax (__init__.py, line 151)
CRITICAL: RuntimeError: Error encountered
```

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/api_domain_order/guide.de-de.md
  | SyntaxError: invalid syntax (__init__.py, line 151)
CRITICAL: RuntimeError: Error encountered
```

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image 3948.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3949.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3936.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3953.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Other images have empty alt attributes
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.de-de.md
  | ComposerError: expected a single document in the stream
  in "<unicode string>", line 1, column 1:
    title: "PHP-Version eines Webhos ... 
    ^
but found another document
  in "<unicode string>", line 6, column 1:
    --- 
    ^
CRITICAL: RuntimeError: Error encountered
```

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image 3948.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3949.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3936.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3953.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Other images have empty alt attributes
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.de-de.md
  | ComposerError: expected a single document in the stream
  in "<unicode string>", line 1, column 1:
    title: "PHP Version eines Webhos ... 
    ^
but found another document
  in "<unicode string>", line 6, column 1:
    --- 
    ^
CRITICAL: RuntimeError: Error encountered
```

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image 3948.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3949.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3936.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3953.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Other images have empty alt attributes
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.de-de.md
  | ComposerError: expected a single document in the stream
  in "<unicode string>", line 1, column 1:
    title: "PHP Version eines Webhos ... 
    ^
but found another document
  in "<unicode string>", line 6, column 1:
    --- 
    ^
CRITICAL: RuntimeError: Error encountered
```

File: pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-es.md [hunk 1]
```diff
@@ -1,73 +1,102 @@
...
 
 ## Requisitos
 
-- Tener contratado un [plan de hosting de OVH](https://www.ovhcloud.com/es-es/web-hosting/){.external} (salvo los planes de hosting Cloud Web).
-- Según el método utilizado, tener acceso a la gestión del plan de hosting desde el [área de cliente de OVH](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) o disponer de las claves necesarias para conectarse al espacio de almacenamiento. 
+- Tener contratado un [plan de hosting de OVHcloud](https://www.ovhcloud.com/es-es/web-hosting/){.external}, a excepción de un plan de hosting Cloud Web.
+- Tener acceso a su plan de hosting desde el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) o conocer la información necesaria para conectarse al[espacio de almacenamiento FTP](https://docs.ovh.com/es/hosting/conexion-espacio-almacenamiento-ftp-alojamiento-web/). 
 
-## Procedimiento
+## En la práctica
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-es.md]:
```suggestion
## Procedimiento
```

File: pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-es.md [hunk 2]
```diff
@@ -1,73 +1,102 @@
...
 
-- **Utilizando el asistente de configuración del área de cliente de OVH**:  Esta opción es menos técnica y requiere estar conectado al área de cliente, desde donde podrá elegir la versión de PHP que desee. Para más información, consulte nuestra guía [Modificar la configuración de un alojamiento web](https://docs.ovh.com/es/hosting/cambiar_el_entorno_de_ejecucion_de_un_alojamiento/){.external}.
+- **Modificando manualmente un archivo en su espacio de almacenamiento** : esta solución es más técnica y necesita estar conectado a su [espacio de almacenamiento FTP](https://docs.ovh.com/es/hosting/conexion-espacio-almacenamiento-ftp-alojamiento-web/), donde deberá editar el archivo.ovhconfig`. Para más información, consulte nuestra guía ["Configurar el archivo .ovhconfig de un alojamiento web"](https://docs.ovh.com/es/hosting/configurar-archivo-ovhconfig/){.external} .
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-es.md]:
```suggestion
- **Modificando manualmente un archivo en su espacio de almacenamiento** : esta solución es más técnica y necesita estar conectado a su [espacio de almacenamiento FTP](https://docs.ovh.com/es/hosting/conexion-espacio-almacenamiento-ftp-alojamiento-web/), donde deberá editar el archivo `.ovhconfig`. Para más información, consulte nuestra guía ["Configurar el archivo .ovhconfig de un alojamiento web"](https://docs.ovh.com/es/hosting/configurar-archivo-ovhconfig/){.external} .
```

File: pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-es.md [hunk 3]
```diff
@@ -1,73 +1,102 @@
...
 
-- **Modificando manualmente un archivo en el espacio de almacenamiento**: Esta opción es más técnica y requiere estar conectado al espacio de almacenamiento, desde donde deberá editar el archivo **.ovhconfig**. Para más información, consulte la guía [Configurar el archivo .ovhconfig de un alojamiento web](https://docs.ovh.com/es/hosting/configurar-archivo-ovhconfig/){.external}.
+> [!primary]
+>
+> La modificación de la versión de PHP a través de un archivo ".htaccess" ya no es posible en los últimos planes de [hosting de OVHcloud](https://www.ovhcloud.com/es-es/web-hosting/){.external}.<br>
+> El comando que permite cambiar la versión de PHP en el archivo ".htaccess" no permite utilizar las versiones recientes de PHP en nuestras infraestructuras.<br>
+> Para ello, deberá utilizar obligatoriamente el archivo.`ovhconfig` con ayuda de nuestra documentación ["Configurar el archivo .ovhconfig de mi alojamiento web"](https://docs.ovh.com/es/hosting/configurar-archivo-ovhconfig/){.external}.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-es.md]:
```suggestion
> Para ello, deberá utilizar obligatoriamente el archivo `.ovhconfig` con ayuda de nuestra documentación ["Configurar el archivo .ovhconfig de mi alojamiento web"](https://docs.ovh.com/es/hosting/configurar-archivo-ovhconfig/){.external}.
```

File: pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-us.md [hunk 4]
```diff
@@ -1,73 +1,102 @@
...
 
 ## Requisitos
 
-- Tener contratado un [plan de hosting de OVHcloud](https://www.ovhcloud.com/es/web-hosting/){.external} (salvo los planes de hosting Cloud Web).
-- Según el método utilizado, tener acceso a la gestión del plan de hosting desde el [área de cliente de OVHcloud](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/world/&ovhSubsidiary=ws) o disponer de las claves necesarias para conectarse al espacio de almacenamiento. 
+- Tener contratado un [plan de hosting de OVHcloud](https://www.ovhcloud.com/es/web-hosting/){.external}, a excepción de un plan de hosting Cloud Web.
+- Tener acceso a su plan de hosting desde el [área de cliente de OVHcloud](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/world/&ovhSubsidiary=ws) o conocer la información necesaria para conectarse al[espacio de almacenamiento FTP](https://docs.ovh.com/us/es/hosting/conexion-espacio-almacenamiento-ftp-alojamiento-web/). 
 
-## Procedimiento
+## En la práctica
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-us.md]:
```suggestion
## Procedimiento
```

File: pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-us.md [hunk 5]
```diff
@@ -1,73 +1,102 @@
...
 
-- **Utilizando el asistente de configuración del área de cliente de OVHcloud**:  Esta opción es menos técnica y requiere estar conectado al área de cliente, desde donde podrá elegir la versión de PHP que desee. Para más información, consulte nuestra guía [Modificar la configuración de un alojamiento web](../cambiar_el_entorno_de_ejecucion_de_un_alojamiento/){.external}.
+- **Modificando manualmente un archivo en su espacio de almacenamiento** : esta solución es más técnica y necesita estar conectado a su [espacio de almacenamiento FTP](https://docs.ovh.com/us/es/hosting/conexion-espacio-almacenamiento-ftp-alojamiento-web/), donde deberá editar el archivo.ovhconfig`. Para más información, consulte nuestra guía ["Configurar el archivo .ovhconfig de un alojamiento web"](https://docs.ovh.com/us/es/hosting/configurar-archivo-ovhconfig/){.external} .
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-us.md]:
```suggestion
- **Modificando manualmente un archivo en su espacio de almacenamiento** : esta solución es más técnica y necesita estar conectado a su [espacio de almacenamiento FTP](https://docs.ovh.com/us/es/hosting/conexion-espacio-almacenamiento-ftp-alojamiento-web/), donde deberá editar el archivo `.ovhconfig`. Para más información, consulte nuestra guía ["Configurar el archivo .ovhconfig de un alojamiento web"](https://docs.ovh.com/us/es/hosting/configurar-archivo-ovhconfig/){.external} .
```

File: pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-us.md [hunk 6]
```diff
@@ -1,73 +1,102 @@
...
 
-- **Modificando manualmente un archivo en el espacio de almacenamiento**: Esta opción es más técnica y requiere estar conectado al espacio de almacenamiento, desde donde deberá editar el archivo **.ovhconfig**. Para más información, consulte la guía [Configurar el archivo .ovhconfig de un alojamiento web](../configurar-archivo-ovhconfig/){.external}.
+> [!primary]
+>
+> La modificación de la versión de PHP a través de un archivo ".htaccess" ya no es posible en los últimos planes de [hosting de OVHcloud](https://www.ovhcloud.com/es/web-hosting/){.external}.<br>
+> El comando que permite cambiar la versión de PHP en el archivo ".htaccess" no permite utilizar las versiones recientes de PHP en nuestras infraestructuras.<br>
+> Para ello, deberá utilizar obligatoriamente el archivo.`ovhconfig` con ayuda de nuestra documentación ["Configurar el archivo .ovhconfig de mi alojamiento web"](https://docs.ovh.com/us/es/hosting/configurar-archivo-ovhconfig/){.external}.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.es-us.md]:
```suggestion
> Para ello, deberá utilizar obligatoriamente el archivo `.ovhconfig` con ayuda de nuestra documentación ["Configurar el archivo .ovhconfig de mi alojamiento web"](https://docs.ovh.com/us/es/hosting/configurar-archivo-ovhconfig/){.external}.
```

File: pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.pt-pt.md [hunk 7]
```diff
@@ -1,73 +1,102 @@
...
+- **através de um assistente de configuração a partir da sua Área de Cliente**: uma vez ligado ao seu [Área de Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt), poderá escolher a nova versão de PHP pretendida entre outros parâmetros. Consulte o nosso manual ["Alterar a configuração do alojamento web"](https://docs.ovh.com/pt/hosting/modificar_o_ambiente_de_execucao_do_meu_alojamento_web/){.external} para realizar esta operação.
+
+- **alterando manualmente um ficheiro no seu espaço de armazenamento**: esta solução é mais técnica e precisa de aceder ao seu [espaço de armazenamento FTP](https://docs.ovh.com/pt/hosting/aceder-espaco-de-armazenamento-ftp-alojamento-web/) onde deverá modificar o ficheiro ^.ovhconfig`. Consulte o nosso manual [Configurar o ficheiro .ovhconfig do alojamento web](https://docs.ovh.com/pt/hosting/configurar-ficheiro-ovhconfig/){.external} para o fazer.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/php_configure_php_on_your_web_hosting_2014/guide.pt-pt.md]:
```suggestion
- **alterando manualmente um ficheiro no seu espaço de armazenamento**: esta solução é mais técnica e precisa de aceder ao seu [espaço de armazenamento FTP](https://docs.ovh.com/pt/hosting/aceder-espaco-de-armazenamento-ftp-alojamento-web/) onde deverá modificar o ficheiro `.ovhconfig`. Consulte o nosso manual [Configurar o ficheiro .ovhconfig do alojamento web](https://docs.ovh.com/pt/hosting/configurar-ficheiro-ovhconfig/){.external} para o fazer.
```

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]

**Review by OVH-Team-Guides — APPROVED** [review]

**Pull request merged**